.github/workflows/cicd.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           java-version: '12.x'
       - uses: subosito/flutter-action@v1
         with:
-          channel: 'beta'
+          channel: 'stable'
```

Comment:
Changed our CI to stable because apparently on beta, something changed on formatter tool, which was causa to the analyze tool work fine on local, but break on CI.

Since web and even desktop is on stable now, I think it is safe for us to use stable on CI.

Reply:
And as our doc states:
> Flame keeps it support on the stable channel. The dev, beta and master channel should work, but we don't support them. This means that issues happening outside the stable channel are not a priority.

We don't have to test for it anymore :-)

Reply:
But it might be interesting to have a `cicd-beta.yml` with jobs that are allowed to fail to see if we have to prepare thing for the next release.